lib/client-api/src/StoryStoreFacade.ts

```diff
@@ -23,8 +23,6 @@ import {
   StoryIndexEntry,
 } from '@storybook/store';
 
-const { STORIES = [] } = global;
```

Comment:
We can't read this right away because it doesn't get set until after `require` in storyshots 🤮

Reply:
tbf I think this is actually an improvement.. when accessing things from the global scope, you want to access them JIT, to prevent memory leaks?

If global.STORIES changes over time, this module keeps a reference around of the first one for ever... the new code doesn't.

Reply:
I guess so, although why would it change over time? (Apart from this reason ;) )